libsubmit/providers/aws/aws.py

```diff
@@ -414,8 +416,16 @@ def security_group(self, vpc):
                 'IpRanges': [{
                     'CidrIp': '0.0.0.0/0'
                 }],
+            }, {
```

Comment:
Great catch! This was blocking all incoming ssh connections. It looks like we also need to update the outgoing port ranges, I'll make those changes once these are merged.